rosdep/python.yaml

```diff
@@ -5799,6 +5799,12 @@ python3-gi:
   fedora: [python3-gobject]
   gentoo: [dev-python/pygobject]
   ubuntu: [python3-gi]
+python3-git:
+  arch: [python-gobject]
```

Comment:
Is this really the right package for arch?  I would guess that this is `python-gitpython` or something like that, but I'm not sure.

Reply:
I feel you. That looks a bit strange indeed. For Py2 version that was added in https://github.com/ros/rosdistro/pull/16946 it's the same package. As in my comment in this PR, I couldn't find an info about Py2 and 3 differentiation for Arch. I'm not entirely sure.

Reply:
I somehow missed that when I reviewed #16946 or I would have asked there. I'm not sure why the GObject bindings for python would include the gitpython module so that seems either like a mistake or a very unusual circumstance. In any case, the Arch Linux package https://www.archlinux.org/packages/community/any/python-gitpython/ is the one that matches the library being provided on other platforms so I think this should change to use that.

Reply:
Makes sense. Then I pushed 2 commits, one for updating Py3 Arch, and the other for fixing Py2 Arch.